### DISCUSSION TITLE
[Résolu] Besoin d'aide pour la configuration de la database

### DISCUSSION THREAD

**Discussion by Ouragann** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#23] [Q&A] [answered]
Bonjour,
Je débute sur tout : Github, MariaDB.

Comme requis j'ai installer MariaDB, j'ai créer une BBD avec les paramètres suivants : 
https://puu.sh/Juz9q/05135215ea.png

Mais dès que je veux "migrer la BDD" j'ai l'erreur suivante :

"C:\FXServer\server\resources\[soz]\soz-core>yarn run prisma migrate deploy
yarn run v1.22.19
$ C:\FXServer\server\resources\[soz]\soz-core\node_modules\.bin\prisma migrate deploy
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "undefined" at "127.0.0.1:3306"

Error: P3004: The `mysql` database is a system database, it should not be altered with prisma migrate. Please connect to another database.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command."

Je pense que les crédentials de la BDD ne doivent pas être bon dans le .env 

DATABASE_URL="mysql://root:root@127.0.0.1:3306?charset=utf8mb4"

# Mysql connection string
set mysql_connection_string "mysql://root:root@127.0.0.1:3306?charset=utf8mb4"
set mysql_debug true

EDIT : J'ai tenter de creer la base de donnée soz ou db mais rien ne change.

J'ai tenter de faire des recherches mais sans succès, c'est pour cela que je viens demander de l'aide, merci d'avance si vous pouvez m'aider

**Comment by Alithegood** [comment]
tu as bien modifier le env.cfg ? en plus du .env

**Comment by Ouragann** [comment, marked as answer]
J'ai résolu en recopiant la base des url de sql :

mysql://soz:soz@localhost/soz il faut creer une base de donnée soz dans mariaBD

J'ai remplacer par root pour moi
